src/file.ts

```diff
@@ -3246,7 +3246,7 @@ class File extends ServiceObject<File> {
    * @see [Storage Classes]{@link https://cloud.google.com/storage/docs/storage-classes}
    *
    * @param {string} storageClass The new storage class. (`standard`,
-   *     `nearline`, or `coldline`)
+   *     `nearline`, `archive`, or `coldline`)
```

Comment:
I would order these as standard, nearline, coldline, archive (archive is the coldest).